lib/teamsnap.rb

```diff
@@ -34,6 +35,15 @@ def init(opts = {})
       ##   create universally accessible TeamSnap.root_client
       self.root_client = TeamSnap::Client.new(:token => token)
 
+      ##   include any feature headers
+      if opts[:header]
```

Comment:
Yeah, multiple headers are supported.  It sounds like the name of the variable is confusing.  I'll update that.